Common/cpp/Fabric/PropsRegistry.cpp

```diff
@@ -21,6 +21,12 @@ void PropsRegistry::update(
     // no need to update `.first` because ShadowNodeFamily doesn't change
     // merge new props with old props
     it->second.second.update(props);
+
+    // Update ShadowNode stored in the map in case it was replaced
```

Comment:
I'm okay with this change but maybe it would be faster to always assign a new pair? This way we can remove most of the code here 😄 
```cpp
void PropsRegistry::update(
    const ShadowNode::Shared &shadowNode,
    folly::dynamic &&props) {
  map_[tag] = std::make_pair(shadowNode, props);
}
```

Reply:
Turns out we can't do this because we still need to call `.update()` on `folly::dynamic`.